Timesheets kbr (#568)
odoo/documentation-user: timesheets/overview/get_started.rst

===========================
Get Started with Timesheets
===========================

> When you timesheet you make sure your project is profitable, as it allows you to track the time
> spent on tasks accurately, and to make fair comparisons with the forecasted time. In addition to
> that, you become more proactive, as you can do it it from any device, anywhere.

Reviewer: ```suggestion
Filling up timesheets allows tracking the time spent on tasks, but also making fair comparisons with the forecasted time while ensuring the project's profitability. In addition to that, you become more proactive, as you can do it from any device, anywhere.
```

Author: What do you think now?

Choose an encoding unit
=======================

> | Under :menuselection:`Timesheets --> Configuration --> Settings` choose the unit of measure to
>   register your timesheet. This provides tools and widgets to help you encoding.

Reviewer: ```suggestion
| Under :menuselection:`Timesheets --> Configuration --> Settings`, choose the unit of measure to
  register your timesheet. This provides tools and widgets to help you encoding.
```

| *Minimal duration* is the minimum time at which a task is recorded.
| *Rounding up* allows you to time up or down to the nearest minutes or hours. For example: if the
  interval is set to 30min, a 14min entry will show up in reports as 0min, and a 29min entry

>   as 30mi.

Reviewer: ```suggestion
  as 30min.
```

.. image:: media/get1.png
   :align: center
   :alt: Set the time unit used to record your timesheets in Odoo Timesheets application

.. note::
   Reportings are expressed in hours, the default value.

Create a task from a SO & timesheet on it
=========================================

> | Under the *General Settings* of your product, categorize it as a service.
> | On the *Sales* tab, choose if you want to invoice customers based on the quantity previously
>   ordered (the one sent on a quotation, for instance), on the time recorded while the service was
>   happening, or by manually adding the time spent when creating the invoice.
> | Chose the right *Service Tracking* for you, and from now on, when a sales order is confirmed,
>   Odoo automatically creates a task/project (if that was the option chosen) and allows you to
>   timesheet on it.

Reviewer: ```suggestion
| Under the *General Settings* of your product, categorize it as a service.
| On the *Sales* tab, choose if you want to invoice customers based on the quantity previously
  ordered (the one sent on a quotation, for instance), on the time recorded while the service was happening, or by manually adding the time spent when creating the invoice.
| Chose the right *Service Tracking* for you and, from now on, when a sales order is confirmed,
  Odoo automatically creates a task/project (if that was the chosen option) and allows you to timesheet on it.
```

Author: I will not remove that part as it is an option (follow img) users have (on the time recorded (while the service was being executed)). It is correct and relevant (:
I did change a couple of words though. It might sound better now.

Reviewer: I didn't ask you to remove that part. I just made some changes. Maybe I miss clicked somewhere or something...

Author: oh gosh.. you are right. I am sorry..and I went through the paragraph more than once to make sure it wasn't there anymore ^^ Sorry, Ludvig. If could please just read it one last time, I'd appreciate it (:

Reviewer: You can find the suggestion below :+1:

===========================
Get Started with Timesheets
===========================

> Filling up timesheets allows you to accurately track the time spent on tasks, making sure your

Reviewer 2: I would flesh this out a bit more. 

I'd say something about timesheet allows you to bill for the amount of work spent, not more, not less, that you can add description of what you worked on. I'd split the last part with the devices to another line.

> Filling up timesheets allows you to accurately track the time spent on tasks making sure your
> project is profitable. It also allows you to make fair comparisons with forecasted times and to
> become more proactive, as you can do it it from any device, anywhere.

Reviewer: ```suggestion
Filling up timesheets allows you to accurately track the time spent on tasks, making sure your
project is profitable. It also allows you to make fair comparisons with forecasted times and to
become more proactive, as you can do it from any device, anywhere.
```

Reviewer: You forgot an "it" ^^ @kbr-odoo

Choose an encoding unit
=======================

| Under :menuselection:`Timesheets --> Configuration --> Settings`, choose the unit of measure to
  register your timesheet. This provides tools and widgets to help you encoding.
| *Minimal duration* is the minimum time at which a task is recorded.
| *Rounding up* allows you to time up or down to the nearest minutes or hours. For example: if the
  interval is set to 30min, a 14min entry will show up in reports as 0min, and a 29min entry
  as 30min.

.. image:: media/get1.png
   :align: center
   :alt: Set the time unit used to record your timesheets in Odoo Timesheets application

.. note::
   Reportings are expressed in hours, the default value.

Create a task from a SO & timesheet on it
=========================================

> | Under the *General Settings* of your product, categorize it as a service.
> | On the *Sales* tab, (section *Sales Invoicing Policy*) choose if you want to invoice customers
>   based on the quantity previous ordered (the one sent on a quotation, for instance), on the time
>   recorded (while the service was being executed), or by manually adding the time spent when
>   creating the invoice.
> | Chose the right *Service Tracking* for you and, from now on, when a sales order is confirmed,
>   Odoo automatically creates a task/project (if that was the chosen option) and allows you to
>   timesheet on it.

Reviewer: ```suggestion
| Under the *General Settings* of your product, categorize it as a service.
| On the *Sales* tab, (section *Sales Invoicing Policy*) choose if you want to invoice customers
  based on the quantity previously ordered (the one sent on a quotation, for instance), on the time recorded (while the service was being executed), or by manually adding the time spent when creating the invoice.
| Choose the right *Service Tracking* for you and, from now on, when a sales order is confirmed,
  Odoo automatically creates a task/project (if that was the chosen option) and allows you to timesheet on it.
```

Reviewer: I think I take parts that are too big... so it doesn't show what are the changes I suggest...
